airbyte-integrations/connectors/destination-azure-blob-storage-python/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM airbyte/python-connector-base:1.1.0 AS base
+FROM airbyte/python-connector-base:2.0.0 AS base
```

Comment:
Is this just an airbyte upgrade?

Reply:
Yep. Details on [what they changed on each base image](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/base_images/README.md) are not all that comprehensive.. but the latest base image does work.